browsers/samsunginternet_android.json

```diff
@@ -291,12 +291,13 @@
         },
         "26.0": {
           "release_date": "2024-06-07",
-          "status": "current",
+          "status": "retired",
           "engine": "Blink",
           "engine_version": "122"
         },
         "27.0": {
-          "status": "beta",
+          "release_date": "2024-09-08",
+          "status": "current",
```

Comment:
It appears that we haven't had a single visit of MDN with Samsung Internet 27.0, indicating to me that it is actually not yet released.